pegasus_wrapper/workflow.py

```diff
@@ -195,6 +213,51 @@ def get_file(self, logical_file_name: str) -> File:
             )
         return self._lfn_to_file[logical_file_name]
 
+    def _define_transformation(
+        self,
+        name: str,
+        pfn: str,
+        *,
+        namespace: Optional[str] = None,
+        version: Optional[str] = None,
+        site: Optional[str] = None,
+        is_stageable: bool = False,
+        bypass_staging: bool = True,
+        container: Optional[Container] = None,
+        arch: Optional[Arch] = None,
+        os_type: Optional[OS] = None,
+    ) -> PegasusTransformation:
+        # Try to see if we have the target transformation already made
+        if name in self._transformation_name_to_transformations:
+            for transformation in self._transformation_name_to_transformations[name]:
```

Comment:
You could simplify this into a one-liner like:
```suggestion
            for transformation in self._transformation_name_to_transformations.get(name, default=[]):
```